renderdoc/driver/gl/wrappers/gl_emulated.cpp

```diff
@@ -2921,53 +2938,47 @@ void APIENTRY _glGetTexImage(GLenum target, GLint level, const GLenum format, co
     GL.glColorMask(GL_TRUE, GL_TRUE, GL_TRUE, GL_TRUE);
 
     GL.glActiveTexture(eGL_TEXTURE0);
-    GL.glBindTexture(eGL_TEXTURE_2D, boundTexture);
+    GL.glBindTexture(target, boundTexture);
 
     GLuint prog;
 
     {
       const char *vs =
-          "attribute vec2 pos;\n"
+          "#version 300 es\n"
+          "in vec2 pos;\n"
```

Comment:
This will fail if we're not capturing on GLES 3.0. It's fine to only support that for the 2D Array/3D case (I don't think they were supported in 2.0), but this codepath still needs to function on GLES 2.0.

You should make the higher declared GLSL version conditional on whether we're doing a non-2D texture.

Reply:
Yes, it was question from me, how it will works on gles 2.0 environment, because I can't test it. So I left both implementations, 2d for gles 2.0/3.0, 3d/arrays will work only for gles 3.0